### PR TITLE
Minor Corrections

### PR DESCRIPTION
**Pull request by mfarif**
a) Updated "UF_LIRA" to "UFLIRA"  to correct the test_smtlibscript.py
b) Updated minor change to correct two DeprecationWarnings.

### CODE REVIEW

File: pysmt/smtlib/parser/__init__.py [hunk 1]
```diff
@@ -80,7 +80,7 @@
     # no ambiguity when importing the parser: the only way to load the
     # cython version is by the so_path that targets .pyxbld .
     #
-    import imp
+    import implib
```
Comment (mikand) [inline pysmt/smtlib/parser/__init__.py:83]: This fix does not seem to work with cython: https://dev.azure.com/micheliandrea/PySMT/_build/results?buildId=548&view=logs&jobId=5b3fdf3a-600d-577c-6029-5dbd4ada42e3&j=5b3fdf3a-600d-577c-6029-5dbd4ada42e3&t=4c81876a-3912-5544-1861-6e026cdb91ca

**Review by mikand — CHANGES_REQUESTED** [review]
@mfarif Thanks for this! There seems to be a problem with the tests in environments where cython is installed.

Also, please add your name and email to the CONTRIBUTORS file to signal that you accept the Contributor License Agreement: https://github.com/pysmt/pysmt/blob/master/docs/development.rst#licensing

File: pysmt/smtlib/parser/__init__.py (hunk 1 above)
Comment (mfarif) [reply pysmt/smtlib/parser/__init__.py]: Done

**Comment by mfarif** [comment]
Resolved all the problems with the build. 
Thank you :)

**Comment by farif** [comment]
Andrea CI/CD compiled ok. When do you consider merging it?

**Comment by mikand** [comment]
Hi @mfarif, I can merge it right away, I was waiting just because you left the "draft" tag (you need to click the "Ready for review" button next time).

Thanks a lot!

**Review by mikand — APPROVED** [review]

**Pull request merged**